x-pack/plugins/fleet/public/applications/fleet/constants/page_paths.ts

```diff
@@ -51,8 +51,8 @@ export const PAGE_ROUTING_PATHS = {
   fleet: '/fleet',
   fleet_agent_list: '/fleet/agents',
   fleet_agent_details: '/fleet/agents/:agentId/:tabId?',
-  fleet_agent_details_events: '/fleet/agents/:agentId',
-  fleet_agent_details_details: '/fleet/agents/:agentId/details',
+  fleet_agent_details_details: '/fleet/agents/:agentId',
```

Comment:
do we need that? looks like you are using `fleet_agent_details` for routing after

Reply:
removed!